editor/src/clj/editor/sprite.clj

```diff
@@ -293,7 +293,7 @@
               (if unassigned-texture-error
                 [unassigned-texture-error]
                 [(validation/prop-error :fatal _node-id :textures shader/page-count-mismatch-error-message is-paged-material texture-page-count material-max-page-count label)
-                 (when (nil? unassigned-default-animation-error)
+                 (when (and anim-data (nil? unassigned-default-animation-error))
```

Comment:
If anim-data is nil, that means that the referenced atlas failed to compile, and we are going to get a compilation error there when building. Making an animation missing error here is unnecessary at this point.